<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAddAttrDialogBase</class>
 <widget class="QDialog" name="QgsAddAttrDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Field</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>N&amp;ame</string>
     </property>
     <property name="buddy">
      <cstring>mNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mNameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>Comment</string>
     </property>
     <property name="buddy">
      <cstring>mCommentEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mCommentEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="buddy">
      <cstring>mTypeBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mTypeBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Provider type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="mTypeName">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="buddy">
      <cstring>mTypeBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mLengthLabel">
     <property name="toolTip">
      <string>Total length of field (including the number of digits after the decimal place for decimal fields).&lt;br&gt;For example 123.45 requires a decimal field length of 5, and 123456 requires an integer field length of 6.</string>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
     <property name="buddy">
      <cstring>mLength</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="mLength">
     <property name="toolTip">
      <string>Total length of field (including the number of digits after the decimal place for decimal fields).&lt;br&gt;For example 123.45 requires a decimal field length of 5, and 123456 requires an integer field length of 6.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mPrecLabel">
     <property name="toolTip">
      <string>Maximum number of digits after the decimal place. For example 123.45 requires a field precision of 2.</string>
     </property>
     <property name="text">
      <string>Precision</string>
     </property>
     <property name="buddy">
      <cstring>mPrec</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="mPrec">
     <property name="toolTip">
      <string>Maximum number of digits after the decimal place. For example 123.45 requires a field precision of 2.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mNameEdit</tabstop>
  <tabstop>mCommentEdit</tabstop>
  <tabstop>mTypeBox</tabstop>
  <tabstop>mLength</tabstop>
  <tabstop>mPrec</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsAddAttrDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsAddAttrDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
